transaction-view/src/resolved_transaction_view.rs

```diff
@@ -113,7 +113,7 @@ impl<D: TransactionData> ResolvedTransactionView<D> {
             };
 
             // If the key is reserved it cannot be writable.
-            is_writable_cache.push(is_requested_write && !reserved_account_keys.contains(key));
+            is_writable_cache[index] = is_requested_write && !reserved_account_keys.contains(key);
```

Comment:
Can you add a comment that a sanitized transaction is guaranteed to not have more than 256 accounts?